src/index.ts

```diff
@@ -0,0 +1,5 @@
+import usePromise from './usePromise'
```

Comment:
Instead of introducing a new file which will bloat the total size of the library, I'd rather rename `usePromise.ts` to `index.ts`. That said, this is a perfectly valid package definition and if an IDE doesn't support it that should be reported as a bug for the IDE, and not as a change in a valid library.

Did you submit a bug report to the author of your IDE? Or has someone perhaps already reported this as an issue?

Reply:
It's not an issue with my IDE, probably how I set up typescript config. Let's discard this PR